packages/driver/src/cy/commands/waiting.ts

```diff
@@ -98,15 +113,16 @@ export default (Commands, Cypress, cy, state) => {
         return xhr
       }
 
-      const args: [any, any, any, any, any] = [alias, type, index, num, options]
-
       return cy.retry(() => {
-        return checkForXhr.apply(window, args)
-      }, options)
+        return checkForXhr.apply(window, [alias, type, index, num, options])
+      },
+      // TODO: What should `_log`'s type be?
+      // @ts-expect-error - Incompatible types.
```

Comment:
I think a lot of this has to do with a mismatch between the internal log and external log type. Could be we just update the declaration types but it isn't likely worth the effort since it isn't consumer facing